test_requirements.in

```diff
@@ -2,7 +2,7 @@
 docker~=4.0
 pytest~=5.3
 pytest-xdist~=1.26  # Parallelises the test-runs
-pytest-mock~=1.11
+pytest-mock~=1.13
```

Comment:
you need to rerun the pip-compile to get the test_requirements.txt as well:-)

Reply:
:grimacing: :+1:

Reply:
Interesting... guess that's why... it's already at `pytest-mock==1.13` in `test_requirements.txt.